### DISCUSSION TITLE
闲鱼有人在卖，已举办，未删

### DISCUSSION THREAD

**Discussion by imouup** [mikumifa/biliTickerBuy#292] [聊天专区]
![IMG_8215](https://github.com/mikumifa/biliTickerBuy/assets/61614535/35c7d001-242c-43cb-a8bb-f1dcf49b84a9)
![IMG_8216](https://github.com/mikumifa/biliTickerBuy/assets/61614535/071071f9-35b4-46f8-bb3f-1c3fdf98e0e7)

**Comment by mikumifa** [comment]
我今天已经劝删很多了，没办法太多了

**Reply by raven-114** [reply]
这种人纯纯的有病

**Comment by ChrisZ-123** [comment]
还是没删，受害者冒个泡

**Reply by imouup** [reply]
诶

**Comment by Nokoyoku** [comment]
咱开源是给他这样用的？😡

**Comment by lemonumi** [comment]
看到个已经卖了5000多了

**Comment by 1321312424124** [comment]
这个情况我也举报过了，但是还是不删
